pipeline/pipeline/backend.py

```diff
@@ -133,7 +133,7 @@ def run(self, pipeline, dry_run, verbose, delete_scratch_on_exit):  # pylint: di
         subdir_name = 'pipeline-{}'.format(uuid.uuid4().hex[:12])
```

Comment:
Ultimately, we want to use the scratch bucket specified by the user -- correct?

Reply:
yes